keadm/app/cmd/common/types.go

```diff
@@ -250,3 +305,56 @@ type EdgeYamlSt struct {
 	EdgeHub EdgeHubSt  `yaml:"edgehub"`
 	EdgeD   EdgeDSt    `yaml:"edged"`
 }
+
+// CreateWebSocketUrl create websocket url
+func CreateWebSocketUrl(address string, port uint16, id string) string {
+	return fmt.Sprintf("wss://%s:%v/e632aba927ea4ac2b575ec1603d56f10/%s/events", address, port, id)
+}
+
+func NewEdgeYamlSt() *EdgeYamlSt {
+	edgeID := "fb4ebb70-2783-42b8-b3ef-63e2fd6d242e"
+	url := CreateWebSocketUrl("0.0.0.0", uint16(DefaultWebSocketPort), "fb4ebb70-2783-42b8-b3ef-63e2fd6d242e/")
+	version := "2.0.0"
```

Comment:
we need to get the version from the release package ? I am not which branch or release does "2.0.0" represent.

Reply:
@samy2019 need your help.